src/Infrastructure/Repositories/EmployeeRepository.cs

```diff
@@ -413,19 +413,74 @@ public async Task AddOrUpdateEmployeeInformation(string employeeEmail, Emergency
     public async Task<List<ProjectExperience>> GetProjectExperiencesByEmailAndCompetencies(string email,
         List<string> competencies)
     {
+        // Get the employee
         var employeeId = await _db.Employees
             .Where(emp => emp.Email == email)
             .Select(e => e.Id)
             .SingleOrDefaultAsync();
 
-        return await _db.ProjectExperiences
-            .Include(pe => pe.ProjectExperienceRoles)
-            .Include(pe => pe.Competencies)
-            .Where(pe =>
-                pe.EmployeeId == employeeId &&
-                _db.Competencies.Any(c => competencies.Contains(c.Name) && c.ProjectExperienceId == pe.Id))
-            .Select(pe => pe.ToProjectExperience())
-            .ToListAsync();
+        // Setup our ranking of competencies
+        List<(int, string)> competenciesLength = new List<(int, string)>();

```

Comment:
Hvis du definerer en klasse for denne tuppelen (int, string) så er det lettere å skjønne hva som foregår.  String er vel kompetanse og int er års erfaring?